### PR TITLE
WIP: photonize banners

### PR DESCRIPTION
**Pull request by johngruen**
This is still WIP. Basically, I'm pulling the MainInstallButton, Header and Banner stuff into storybook/components. In the process a lot of files are getting moved around/touched. Will finish this up on the plane tomorrow.

### CODE REVIEW

File: content-src/news_updates.yaml [hunk 1]
```diff
@@ -13,8 +13,8 @@
   slug: general-update-4
   title: Page Shot is now Firefox Screenshots
   link: https://medium.com/firefox-test-pilot/screenshots-is-shipping-in-firefox-56-25ecbe0d5b91
-  created: 2017-10-15T12:00:00Z
-  published: 2017-10-15T12:00:00Z
+  created: 2017-10-29T12:00:00Z
```
Comment (johngruen) [inline content-src/news_updates.yaml]: todo: revert this file

File: frontend/src/app/components/Copter/stories.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  )
+  .add('normal', () =>
+    <Copter />
+  )
+  .add('small', () =>
+    <Copter
+     small="true"
+    />
+  )
+  .add('animated', () =>
```
Comment (johngruen) [inline frontend/src/app/components/Copter/stories.js]: todo: actually animate this

File: frontend/src/app/components/UpdateList/index.scss [hunk 3]
```diff
@@ -16,6 +16,8 @@
 
 .update-list-heading {
   font-size: 22px;
+  font-style: italic;
```
Comment (johngruen) [inline frontend/src/app/components/UpdateList/index.scss:19]: todo: delete this

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3031/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/9301f83c65ec4a00f83b7987eb54450f29623138/

File: frontend/src/app/components/MainInstallButton/img/download-icon.svg [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (fzzzy) [inline frontend/src/app/components/MainInstallButton/img/download-icon.svg]: Needs svgo

File: frontend/src/app/components/UpdateList/index.scss (hunk 3 above)
Comment (johngruen) [reply frontend/src/app/components/UpdateList/index.scss:19]: I actually did delete the offending line which was below

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3031/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/03025aefed30eb4de813a3cc0a4056c8c7cb584e/

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3031/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/3b1dbc438208530e6bf7ba9fae971ace18417733/

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3031/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/38d9c93d5cbde46b17aa94036216d16700d442de/

**Comment by lmorchard** [comment]
💥 💥 💥 💥 💥

**Pull request merged**